Security/src/CVE-2023-23397/CVE-2023-23397.ps1

```diff
@@ -1081,7 +1081,37 @@ begin {
 
         Show-Disclaimer @params
 
-        $cleanupCSV = (Import-Csv $CleanupInfoFilePath -Delimiter ',')
+        $importedCsvData = Import-Csv $CleanupInfoFilePath -Delimiter ","
```

Comment:
This isn't the same as what Bill provided here: https://github.com/microsoft/CSS-Exchange/issues/1610#issuecomment-1477836135. 

Why is this different?